File: backend/src/main/java/com/example/backend/models/Query.java
package com.example.backend.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
public class Query {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long userId;
    private Long adsId;
    private int status;

}

File: app/src/main/java/com/example/dropletbarterapp/models/Query.kt
package com.example.dropletbarterapp.models

import com.example.dropletbarterapp.ui.models.StatusConverter
import com.google.gson.annotations.SerializedName

class Query(
    @SerializedName("id") val id: Long,
    @SerializedName("userId") val userId: Long,
    @SerializedName("adsId") val adsId: Long,
    @SerializedName("status") val status: Int,
) {
    var statusEnum = StatusConverter.getStatusByPosition(status)
}